src/main/java/org/springframework/data/couchbase/core/query/Query.java

```diff
@@ -127,6 +128,26 @@ public Query with(final Pageable pageable) {
 		return with(pageable.getSort());
 	}
 
+	/**
+	 * queryScanConsistency
+	 *
+	 * @return queryScanConsistency
+	 */
+	public QueryScanConsistency getConsistency() {
+		return queryScanConsistency;
+	}
+
+	/**
+	 * Sets the given scan consistency on the {@link Query} instance.
+	 *
+	 * @param queryScanConsistency
+	 * @return this
+	 */
+	public Query with(final QueryScanConsistency queryScanConsistency) {
```

Comment:
shouldn't this be more like scanConsistency(..) ? we also name limit etc. explicitly.

Reply:
Ok - I changed it to scanConsistency().  But I was looking at with(Pageable) and with(Sort).
But that pattern breaks down for things like skip and limit which are just int.